static/src/javascripts/projects/common/modules/analytics/beacon.js

```diff
@@ -1,27 +0,0 @@
-import config from 'lib/config';
-
-const fire = (path) => {
-    const img = new Image();
-
-    img.src = config.get('page.beaconUrl') + path;
```

Comment:
should we also remove `beaconUrl` from page configuration?

Reply:
I’m not entirely sure we can, it’s still used in DCR in [several places][beaconUrl], including for Google Analytics:

https://github.com/guardian/dotcom-rendering/blob/44bf079a459f5c29a5c3a03d946bbd28dd5df65a/src/web/browser/ga/ga.ts#L155

[beaconUrl]: https://github.com/guardian/dotcom-rendering/search?q=beaconurl